src/components/layout/sync-button.tsx

```diff
@@ -18,7 +18,7 @@ const SyncButton = () => {
   const { synchronizeApi } = useApi();
   const setError = useSetAtom(errorAtom);
   const yesterday = DateTime.now().minus({ days: 1 });
-  const [syncStartDate, setSyncStartDate] = useState(yesterday);
+  const [syncStartDate, setSyncStartDate] = useState<DateTime<true> | DateTime<false>>(yesterday);
```

Comment:
This causes a TS error, can't it just be DateTime?

Reply:
Setting the type to just DateTime seems to work fine